server/index.js

```diff
@@ -18,11 +18,10 @@ nextApp.prepare().then(() => {
   /* APIs go there */
   expressServer.use('/api', api);
 
-  /*Clean URL for rendering Product Information*/
+  /* Clean URL for rendering Product Information*/
   expressServer.get('/product/:id', (req, res) => {
     const actualPage = '/product';
```

Comment:
I guess you can use req.url or something to get `/product`. More scalable that way.